validator/json/examples/hub-schedule2.json

```diff
@@ -0,0 +1,324 @@
+          }
+        },
+        {
+          "name": "Saal ZIGZAG",
+          "slug": "saal-zigzag",
+          "guid": "62251a07-13e4-5a72-bb3c-8528416ee0f2",
+          "type": "lecturehall",
+          "stream_id": "s3",
+          "capacity": 965,
+          "description": "Lecture hall Z in CCH East, Level +3",
+          "features": {
+            "recording": "record_by_default"
+          },
+          "assembly": {
+            "name": "CCC",
+            "slug": "ccc",
+            "guid": "6840c453-af5c-413c-8127-adcbdcd98e9e"
+          }
+        },
+        {
+          "name": "Stage HUFF",
+          "slug": "stage-huff",
+          "guid": "53f436a0-705b-40eb-974f-2bfce8857b1b",
+          "type": "stage",
+          "stream_id": "s4",
+          "capacity": null,
+          "description": {
+            "en": "Community-Stage in hall H, Level 0",
+            "de": "Community-Bühne in Halle H, EG"
+          },
+          "features": {
+            "recording": "record_by_default"
+          },
+          "assembly": {
+            "name": "CCC",
+            "slug": "ccc",
+            "guid": "6840c453-af5c-413c-8127-adcbdcd98e9e"
+          }
+        },
+        {
+          "name": "Stage YELL",
+          "slug": "stage-yell",
+          "guid": "e58b284a-d3e6-42cc-be2b-7e02c791bf95",
+          "type": "stage",
+          "stream_id": "s5",
+          "capacity": null,
+          "description": {
+            "en": null,
+            "de": "Community-Stage in Saal Y in CCH Ost, Level +2"
+          },
+          "features": {
+            "recording": "record_by_default"
+          },
+          "assembly": {
+            "name": "CCC",
+            "slug": "ccc",
+            "guid": "6840c453-af5c-413c-8127-adcbdcd98e9e"
+          }
+        },
+        …
+      ],
+      "days": [
+        {
+          "index": 1,
+          "date": "2024-12-27",
+          "start": "2024-12-27T09:30:00+00:00",
+          "end": "2024-12-28T06:00:00+01:00"
+        },
+        {
+          "index": 2,
+          "date": "2024-12-28",
+          "start": "2024-12-28T06:00:00+01:00",
+          "end": "2024-12-29T06:00:00+01:00"
+        },
+        {
+          "index": 3,
+          "date": "2024-12-29",
+          "start": "2024-12-29T06:00:00+01:00",
+          "end": "2024-12-30T06:00:00+01:00"
+        },
+        {
+          "index": 4,
+          "date": "2024-12-30",
+          "start": "2024-12-30T06:00:00+01:00",
+          "end": "2024-12-30T17:45:00+00:00"
+        }
+      ]
+    },
+    "events": [
+      {
+        "type": "other",
+        "guid": "270fa44c-821a-577f-8ca3-33e8d4a8c2a7",
+        "start": "2024-12-27T10:30:00+01:00",
+        "end": "2024-12-27T11:00:00+01:00",
+        "duration": "00:30",
+        "room": "Saal 1",
+        "slug": "38c3-38c3-opening-ceremony",
+        "title": "38C3: Opening Ceremony",
+        "subtitle": null,
+        "language": "en",
```

Comment:
Should we rename `language` to `languages`? It is not limited to one language if I remember correctly.

Reply:
Depends if we want to include the languages of possible translations. If this is the language of the talk itself, there can only ever be one i think.

Reply:
I believe talks can still be hold in multiple languages, e.g. where speaker1 talks English and speaker2 talks German.

Reply:
Let’s have one language the talk is held in. If we want translations that should be a different field / indicated in some way.

Reply:
I did not refer to translations here. I really meant a talk which is hold in different languages and is therefore announced as such, e.g. `en, de`.

Reply:
Is there any system that currently supports this? AFAIK neither frab nor pretalx do; pretalx certainly assumes that a talk is in one language.

Reply:
I remember seeing it in the Android app at one point. I would need to investigate to find out what the source backend was.

Reply:
I could be that session with multiple languages originate from a wiki import at 38C3.

Reply:
for self-organized sessions created in the Hub, there is the option to select a ordered list of possible languages, via a dropdown. In general I would see this field as main/original language and add an additional translations field.

Reply:
What does the UI for SOS look like / tell about the language selection? Are they being selected as the speakers languages or as the available translations?